README.md

```diff
@@ -1,102 +1,95 @@
-# lint-prepush    
+# lint-prepush
+
 [![npm version](https://badge.fury.io/js/lint-prepush.svg)](https://www.npmjs.com/package/lint-prepush)
 [![npm downloads](https://img.shields.io/npm/dt/lint-prepush.svg)](https://www.npmtrends.com/lint-prepush)
 [![GitHub license](https://img.shields.io/github/license/theenadayalank/lint-prepush.svg)](https://github.com/theenadayalank/lint-prepush/blob/master/LICENSE)
 [![Build Status](https://travis-ci.org/theenadayalank/lint-prepush.svg?branch=master)](https://travis-ci.org/theenadayalank/lint-prepush)
 
+> Run linters on committed files of a GIT Branch
 
-> Run linters on committed files of a GIT Branch🔬
-
-## Getting Started 🔮
+## Getting Started
 
 This package will run linters on your project for the committed files in your branch.
 
-### Prerequisites🔭
-
-It require Node.js v6 or newer. It also requires a package to manage git hooks. I strongly suggest [Husky](https://github.com/typicode/husky) which I use for most of my projects.
-
-
-```js
-// package.json
-{
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm test",
-      "pre-push": "npm test",
-      "...": "..."
-    }
-  }
-}
-```
-
-### Installing
+### Prerequisites
 
-* This project requires a package to manage git hooks depending. 
-* I strongly suggest [Husky](https://github.com/typicode/husky) which pauses the git push by overriding pre-commit hook (it almost overrides all hooks, here we need only pre-push hook) and allow us to run our custom scripts and resumes pushing.
+- Node.js `>=6.0.0`
```

Comment:
It looks like in the `package.json` we are actually saying `>=7.6.0`.

Given that `7.x.x` isn't a Long Term Support (LTS) version of Node.js, maybe it's worth just jumping to `8.0.0` for the minimum supported version?